capstone/src/main/java/com/google/sps/data/Tag.java

```diff
@@ -98,4 +93,30 @@ private Tag() {
     public static String get(String tag) {
         return tag;
     }
+
+    public static final HashMap<String, Integer> constructUserVectorMap() {
+        HashMap<String, Integer> vector = new HashMap<>();
+        vector.put(Tag.TAG_EDUCATION, 0);
+        vector.put(Tag.TAG_COOKING, 0);
+        vector.put(Tag.TAG_FITNESS, 0);
+        vector.put(Tag.TAG_LITERATURE, 0);
+        vector.put(Tag.TAG_MUSIC, 0);
+        vector.put(Tag.TAG_TECHNOLOGY, 0);
+        vector.put(Tag.TAG_GAMING, 0);
+        vector.put(Tag.TAG_MISCELLANEOUS, 0);
+        vector.put(Tag.TAG_RELIGIOUS, 0);
+        vector.put(Tag.TAG_HISTORIC, 0);
+        vector.put(Tag.TAG_SPACE, 0);
+        vector.put(Tag.TAG_CAREER, 0);
+        vector.put(Tag.TAG_ARCHITECTURE, 0);
+        vector.put(Tag.TAG_ART, 0);
+        vector.put(Tag.TAG_DANCING, 0);
+        vector.put(Tag.TAG_CHALLENGE, 0);
+        vector.put(Tag.TAG_DIETING, 0);
+        vector.put(Tag.TAG_ADVENTURE, 0);
+        vector.put(Tag.TAG_MEDITATION, 0);
+        vector.put(Tag.TAG_NEWS, 0);
```

Comment:
This should be a for loop. As it stands, you have to remember to make a change here if you add a tag. (Also, per a previous PR comment, this will become an enum anyway instead of a bunch of constants)

Reply:
Got it.